fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -37,6 +37,8 @@ statement
         AS queryStatement                                                               #alterView
     | DROP TABLE (IF EXISTS)? qualifiedName FORCE?                                    #dropTable
     | DROP VIEW (IF EXISTS)? qualifiedName                                              #dropView
+    | ADMIN SET FRONTEND CONFIG
+        ('(' property ')')?                                                             #adminSet
```

Comment:
In the old syntax, `admin set frontend config` is also syntax correct, but an error will be reported in Analyzer. This is not a good design, in this position, we can not be compatible with the old syntax. In the new grammar, it is guaranteed that the propery must exist. If it does not exist, a syntax error will be reported in Parser.